app/helpers/application_helper/button/vm_policy.rb

```diff
@@ -2,6 +2,6 @@ class ApplicationHelper::Button::VmPolicy < ApplicationHelper::Button::Basic
   needs_record
 
   def skip?
-    @record.host&.vmm_product.to_s.casecmp("workstation")
+    @record.host.vmm_product.to_s.casecmp("workstation") if @record.host
```

Comment:
Will change, I also do not sure if I understand correct what function must skip from the previous context, so will be nice if someone can confirm if it desired behaviour